### ISSUE TITLE
Compound suggestion thread

### ISSUE THREAD

**Issue by MFernflower** [OpenSourceMalaria/OSM_To_Do_List#440]
Because of the fact that the various ideas me and others have came up with have been deposited in various issues. I feel it is important to get a central thread going to avoid this problem.

**Comment by mattodd** [comment]
It's super important we discuss suggestions. But I'm not so sure about a single Issue - it'll become bloated. Issues are meant to be for things that can be closed in the short term, after a degree of consultation. There are many Issues still open, yes, but the idea here is to have rapid turnover. Longer-term things, which are perhaps archived compounds that are seen as desirable but are not actually being worked on, already have a home on the wiki, for example [here's the section for Series 4](http://openwetware.org/wiki/OpenSourceMalaria:Triazolopyrazine_%28TP%29_Series#Desirable_Compounds_Not_Yet_Synthesised). 

What I'd suggest is that Issues can be used for questions like "How about we make this molecule" _when they are accompanied by reasons for them being made_. Discussion can happen about desirability, and whether anyone has a way of sourcing the compound. After a short period, say a week or two, the Issue can be closed if someone starts making it (which may deserve a new Issue tagged with the orange "Being Synthesised Now") or if there's no enthusiasm, in which case it can be installed on the wiki for posterity, with a link to the relevant discussion (which remains available, crucially, after the Issue is closed).

In some cases we may develop a good plan for some new methodology that takes us towards a new compound - i.e. a self-contained synthetic methodology project. In such cases we can archive a project plan like [these](http://openwetware.org/wiki/OpenSourceMalaria:Synthesis_Branches).

OK? Agreement? Naysayers?

**Comment by MFernflower** [comment]
@mattodd Managed to reconstruct my labnotebook after file host i was using died. Think ill just post various musings I have on there for now

**Comment by drc007** [comment]
It might be better to simply use a google docs spreadsheet. You can then have columns for a brief comment on why the target should be made, who suggested it, any relevant references. Later you can add the name of the chemist assigned to the target, links to questions on this forum about synthesis etc. Eventually when prepared you can add the identifier.

**Comment by MFernflower** [comment]
Closing this issue due to lack of activity

**Comment by mattodd** [comment]
Happy to revisit any time. Google sheet good idea. Current list of
desirable targets is on wiki, but I know there are sign-in barriers for
that.

**Comment by MFernflower** [comment]
@mattodd I registered for the wiki under my real (not pen) name but have
not gotten a reply back yet.. A public spreadsheet seems like a great idea
along with some type of letter code to tell who thought the molecule up.
(Perhaps. MX for my ideas. HA for the Haverford group, CS for swain. Etc).
I also saw all the awesome test results you and Edwin got in and propose a
3-methoxy amide in issue #464
